pkg/agent/openflow/client_test.go

```diff
@@ -80,9 +89,11 @@ func TestIdempotentFlowInstallation(t *testing.T) {
 			client.cookieAllocator = cookie.NewAllocator(0)
 			client.flowOperations = m
 
-			m.EXPECT().Add(gomock.Any()).Return(nil).Times(tc.numAddCalls)
+			m.EXPECT().AddAll(gomock.Any()).Return(nil).Times(1)
+			// Installing the flows successfully at the first time, and all the flows are added into the cache.
 			numCached1, err := tc.installFn(ofClient, tc.cacheKey)
 			require.Nil(t, err, "Error when installing Node flows")
+			assert.Equal(t, tc.numAddCalls, numCached1)
```

Comment:
nit: I suggest renaming `numAddCalls` to `numFlows` since we no longer call `Add` multiple times but instead call `AddAll` one time.